dogs/util/generate-component-styles.js

```diff
@@ -1,9 +1,9 @@
-import mapValues from 'lodash/mapValues'
+import { map } from 'ramda'
 
 export default function generateComponentStyles (styles) {
   return (props) => (renderer) => {
-    return mapValues(styles, (style) => {
+    return map((style) => {
```

Comment:
it'd be ideal if we could curry `styles`, so we only generate the function _once_.

looks like we could use [`R.__`](http://ramdajs.com/docs/#__).

```js
import { map, __ } from 'ramda'

export default function generateComponentStyles (styles) {
  const mapWithStyles = map(__, styles)
  
  return (props) => (renderer) => {
    return mapWithStyles((style) => {
      return renderer.renderRule(style, props)
    })
  }
}
```